plugin/completion.py

```diff
@@ -181,22 +282,23 @@ def format_completion(self, item: dict, can_resolve_completion_items: bool) -> s
 
         lsp_label = item["label"]
         lsp_filter_text = item.get("filterText")
-        lsp_detail = item.get("detail") or ""
-        lsp_detail = html.escape(lsp_detail.replace('\n', ' '))
-
-        if can_resolve_completion_items or "documentation" in item:
-            doc_link = '<a href="subl:lsp_resolve_docs">Documentation</a>'
-        else:
-            doc_link = ''
+        lsp_detail = html.escape(item.get("detail", "").replace('\n', ' '))
 
         if lsp_filter_text and lsp_filter_text != lsp_label:
             st_trigger = lsp_filter_text
             st_annotation = lsp_label
-            st_details = '{} {}'.format(doc_link, lsp_detail) if doc_link else lsp_detail
         else:
             st_trigger = lsp_label
-            st_annotation = lsp_detail
-            st_details = doc_link
+            st_annotation = ""
+
+        st_details = ""
+        if can_resolve_completion_items or "documentation" in item:
+            args = {
+                "item": deep_escape(item)
+            }
+            st_details += "<a href='subl:lsp_resolve_docs {}'>Δ</a>".format(sublime.encode_value(args))
```

Comment:
Hmm, that book symbol might not be supported by many fonts.
📖 emoji works for me but probably won't work in ST on Windows.

Reply:
We can use img tag with data URI.

Reply:
Why not “More...”? It’s shorter than “Documentation”. I’m also honestly fine with the word “Documentation”.

Reply:
If we would want to put more stuff in that field then either "Documentation" or "More" would be kinda confusing as it would blend with other text. Then the image/symbol makes no sense IMO.

But if we are not planning to put anything more in there then I'm fine with "More". It's more generic than "Documentation" and that is IMO good as the values shown after resolving item are not always strictly documentations.

Reply:
"up arrow" is actually the Delta Greek letter `Δ` (I changed it to `More`, so people can understand it easier on what it does) 

before:

![Screenshot from 2020-05-07 22-08-32](https://user-images.githubusercontent.com/22029477/81340163-aaf3c380-90af-11ea-88d2-dc7eb064303e.png)
(I prefer this visually)

after:

![more](https://user-images.githubusercontent.com/22029477/81339326-471ccb00-90ae-11ea-801b-8f58a02ffea7.png)
(I think this is more clearer)

Reply:
I guess it's fine but it does kinda mix with the text (even though it has link style). Maybe have text in square brackets (`[More]`)?

Reply:
this maybe (separated by a `|` pipe)?

![image](https://user-images.githubusercontent.com/22029477/81343102-a251bc00-90b4-11ea-91ef-317432eeb7a0.png)

Reply:
here is how it looks with angle brackets:
![image](https://user-images.githubusercontent.com/22029477/81343330-15f3c900-90b5-11ea-9add-34f7a2377621.png)

Reply:
Pipe WFM

Reply:
I added the pipe,
@rwols feel free to unresolve the conversation if you think otherwise